cob_hardware_config/raw3-1/config/arm_controller.yaml

```diff
@@ -32,4 +32,78 @@ joint_trajectory_controller:
   state_publish_rate:  25
   action_monitor_rate: 10
 
+## position controller
+joint_group_position_controller:
+  type: position_controllers/JointGroupPositionController
+  joints:
+     - arm_shoulder_pan_joint
+     - arm_shoulder_lift_joint
+     - arm_elbow_joint
+     - arm_wrist_1_joint
+     - arm_wrist_2_joint
+     - arm_wrist_3_joint
+  required_drive_mode: 1
+
+arm_1_joint_position_controller:
```

Comment:
convention is to use <joint_name>_position_controller....-> arm_shoulder_pan_joint_position_controller
same for the other controllers...